literature.bib

```diff
@@ -112,34 +121,32 @@ @mastersthesis{Dunkel23
 	author = {Dunkel, Raphael},
 	title = {{One Solver to Rule All Feature Models -- Or Not? Addressing the Algorithm Selection Problem for \#SAT}},
 	school = UniUlm,
-	address = {Germany},
-	type = {Bachelor's Thesis},
+	address = Germany,
+	type = Bachelor,
 	typo3Tags = {SupervisorCS, SupervisorTH, SupervisorTT, FMCounting},
 	month = DEC,
 	year = 2023,
 	note = {To appear}
 }
 
-% Waiting for Oparu
 @mastersthesis{Karrer23,
 	author = {Karrer, Simon},
 	title = {{Interactive Product Configuration for Multiple Feature-Model Versions}},
 	school = UniUlm,
-	address = {Germany},
-	type = {Master's Thesis},
+	address = Germany,
+	type = Master,
 	typo3Tags = {SupervisorTH, SupervisorTT},
 	month = JUL,
 	year = 2023,
 	note = {To appear}
 }
 
-% Waiting for Lukas / Oparu
```

Comment:
why removing these comments?

Reply:
Comments are difficult to handle within the bib file. It is better to include them in dedicated fields. This way they can not accidentally be moved to a different entry, but are always directly tied to one specific entry.

Reply:
true, but then its better to move to comment to a field rather than deleting it I guess :P